templates/default/supermarket.nginx.erb

```diff
@@ -11,7 +11,7 @@ server {
   proxy_set_header        Host            $host;
   proxy_set_header        X-Real-IP       $remote_addr;
   proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
-  proxy_set_header        X-Forwarded-Proto https;
+  proxy_set_header        X-Forwarded-Proto http;
```

Comment:
I might be missing some context here, but if `node['supermarket']['force_ssl']` is `true`, why would we want to set the protocol to `http`?